client/components/checkbox/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import _ from 'lodash';
+
+const Checkbox = ( props ) => {
+	const otherProps = _.omit( props, 'className' );
+	return (
+		<span className={ classNames( props.className, 'form-checkbox', { 'is-disabled': props.disabled } ) } >
```

Comment:
Looks like we'll no longer need the class `form-checkbox` - it relies on Calypso CSS, but can remove it if we're not relying on their FormCheckbox.